custom_components/stromer/sensor.py

```diff
@@ -210,26 +210,13 @@ def __init__(
         self.entity_description = description
         self._attr_unique_id = f"{device_id}-{description.key}"
 
-    @staticmethod
-    def _ensure_timezone(timestamp: datetime | None) -> datetime | None:
-        """Calculate days left until domain expires."""
```

Comment:
Not sure what the stuff about domains and whois means in this context. Maybe it's copy-paste code from a different application? In that case, that other application should probably be checked for wrong timezones as well 🙂

Reply:
I actually don't recall putting this in, but indeed probably borrowed somewhere ... will scratch head in the coming days I guess 👍